src/settings/BatchGroupConfigurationSettings/BatchVoucherExportsList.js

```diff
@@ -25,13 +22,9 @@ const cellFormatters = {
   date: d => (
     <FolioFormattedTime dateString={d.end || d.start} />
   ),
-  exportButton: () => (
-    <Layout element="span" className="full textRight">
-      <IconButton
-        data-test-batch-voucher-export-download
-        icon="download"
-      />
-    </Layout>
+  // eslint-disable-next-line react/prop-types
+  exportButton: ({ batchVoucherId }) => (
+    <BatchVouchersContainer batchVoucherId={batchVoucherId} />
```

Comment:
It's weird component name. If you want to hide button with icon and callback - maybe it makes sense to call component like `ExportVoucherButton` or something